contracts/base/LoansBase.sol

```diff
@@ -554,7 +554,17 @@ contract LoansBase is LoansInterface, Base {
         uint256 loanID,
         uint256 amount,
         address payable recipient
-    ) internal;
+    ) internal {
+        totalCollateral = totalCollateral.sub(amount);
+        loans[loanID].collateral = loans[loanID].collateral.sub(amount);
+        if (collateralToken == _getSettings().ETH_ADDRESS()) {
+            // Ether collateral
+            recipient.transfer(amount);
```

Comment:
here we we unwrap the WETH and then transfer

Reply:
done